### DISCUSSION TITLE
crimping to LRA 32AWG wire?

### DISCUSSION THREAD

**Discussion by PWPInnovator898** [kriswilk/buzzah#3] [General]
Can you describe how you crimped onto the 32AWG LRA wires?  What crimping tool did you use? 
Thanks!
![image](https://user-images.githubusercontent.com/128418916/235799315-68e48061-2935-4ce2-9869-e7802810566e.png)

**Comment by kriswilk** [comment]
I have a kit of Dupont connectors (that's what these are called) that came with a crimp tool. I got mine on Amazon (Canada) but they can be found in many other places online.

https://www.amazon.ca/gp/product/B07R1H3Z8X

However, in this case the wire is so thin it did not crimp reliably. So I ended up just soldering the ends of the wires into the metal dupont receptacle, then crimped it so it could be inserted in the plastic housing.

A really excellent video showing how the connectors are assembled can be found here:

https://www.youtube.com/watch?v=jET1QTP1B7c

**Comment by PWPInnovator898** [comment]
Kris;
Thank you!  That's a clever idea re: just soldering onto the receptacle,
then crimping...
Regards, Phillip

On Wed, May 3, 2023 at 10:14 AM Kris Wilk ***@***.***> wrote:

> I have a kit of Dupont connectors (that's what these are called) that came
> with a crimp tool. I got mine on Amazon (Canada) but they can be found in
> many other places online.
>
> https://www.amazon.ca/gp/product/B07R1H3Z8X
>
> However, in this case the wire is so thin it did not crimp reliably. So I
> ended up just soldering the ends of the wires into the metal dupont
> receptacle, then crimped it so it could be inserted in the plastic housing.
>
> A really excellent video showing how the connectors are assembled can be
> found here:
>
> https://www.youtube.com/watch?v=jET1QTP1B7c
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/kriswilk/buzzah/discussions/3#discussioncomment-5793129>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A6TYIZAAC73JSKET6QS2L6DXEJ76DANCNFSM6AAAAAAXTUS4BY>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Thanks and best regards,
Phillip Callahan

**Reply by kriswilk** [reply]
Yeah, those wires are so very, very thin...it was the only way I could get a reliable connection, especially knowing that these will be moving around some in use...
